Source/Core/DolphinQt/GBAWidget.cpp

```diff
@@ -281,6 +298,10 @@ void GBAWidget::contextMenuEvent(QContextMenuEvent* event)
   unload_action->setEnabled(CanControlCore() && !m_game_title.empty());
   connect(unload_action, &QAction::triggered, this, &GBAWidget::UnloadROM);
 
+  auto* card_action = new QAction(tr("&Load e-Reader Card"), menu);
```

Comment:
Thoughts on changing this to `&Scan e-Reader Card`?

Reply:
Hard agree.  Because most e-Reader cards are in pairs, you have to scan two at once.

Reply:
I've implemented loading multiple cards at once.  Would it make sense to change it to `&Scan e-Reader Card(s)`, or is just `&Scan e-Reader Card` fine?